web/src/engine/websites/Manatoki.ts

```diff
@@ -0,0 +1,71 @@
+import { Tags } from '../Tags';
+import icon from './Manatoki.webp';
+import { Chapter, DecoratableMangaScraper, type MangaPlugin, type Manga } from '../providers/MangaPlugin';
+import { FetchCSS, FetchWindowScript } from '../platform/FetchProvider';
+import * as Common from './decorators/Common';
+import { AddAntiScrapingDetection, FetchRedirection } from './../platform/AntiScrapingDetection';
+
+AddAntiScrapingDetection(async (invoke) => {
+    const result = await invoke<boolean>(`document.querySelector('form[name="fcaptcha"]') && true || false`);
+    return result ? FetchRedirection.Interactive : undefined;
```

Comment:
I would recommend to cleanup the website before it is shown to the user, so just the captcha will be visible without any other distracting elements.

See https://github.com/manga-download/haruneko-proprietary/blob/f3d612d26b978046e5099f2bd36563883f3624ac/haruneko/web/src/engine/websites/CrunchyScan.ts#L21-L37